### PR TITLE
Add BQCorpus Dataset

### PR DESCRIPTION
**Pull request by dyan-dy**
### PR types

database

### PR changes

modified bq_corpus.py according to the opinion

### CODE REVIEW

**Review by ZeyuChen — COMMENTED** [review]
It seems miss a __init__.py in paddlenlp/dataset/

File: paddlenlp/datasets/bq_corpus.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+from paddle.utils.download import get_path_from_url
+from paddlenlp.utils.env import DATA_HOME
+from . import DatasetBuilder
+
+__all__ = ['BQCorpus']
+
+
+class BQCorpus(DatasetBuilder):
+    """
+    BQCorpus: the largest dataset available for for the banking and finance sector
```
Comment (smallv0221) [inline paddlenlp/datasets/bq_corpus.py:15]: Typo here.

File: paddlenlp/datasets/bq_corpus.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    by frozenfish123@Wuhan University
+
+    """
+    lazy = False
+    URL = "https://dataset-bj.cdn.bcebos.com/qianyan/bq_corpus.zip"
+    MD5 = "abe6c480b96cb705b4d24bd522848009"
+    META_INFO = collections.namedtuple('META_INFO', ('file', 'md5'))
+    SPLITS = {
+        'train': META_INFO(
+            os.path.join('BQCorpus', 'train.tsv'),
```
Comment (smallv0221) [inline paddlenlp/datasets/bq_corpus.py:26]: The filepath seems wrong. Have you check your dataset using load_dataset()

**Review by smallv0221 — CHANGES_REQUESTED** [review]
Actually, there is not need to modify init.py to implement new datasets. @ZeyuChen @frozenfish123

File: paddlenlp/datasets/bq_corpus.py [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+from . import DatasetBuilder
+
+__all__ = ['BQCorpus']
+
+
+class BQCorpus(DatasetBuilder):
+    """
+    BQCorpus: the largest dataset available for for the banking and finance sector
+
+    by frozenfish123@Wuhan University
```
Comment (smallv0221) [inline paddlenlp/datasets/bq_corpus.py:17]: Please also give original author information.

**Comment by ZeyuChen** [comment]
> Actually, there is not need to modify init.py to implement new datasets. @ZeyuChen @frozenfish123

Got it.

**Review by smallv0221 — APPROVED** [review]
We decide to merge your dataset and fix it for you due to our release schedule. Excellent work! Thanks for your contribution.

File: paddlenlp/datasets/bq_corpus.py (hunk 3 above)
Comment (dyan-dy) [reply paddlenlp/datasets/bq_corpus.py:17]:
```
"""
    BQ Corpus (Bank Question Corpus), a large-scale domain-specific Chinese corpus for sentence semantic
    equivalence identification is constructed from online Webank custom service logs.

    The BQ corpus contains 120,000 question pairs with manual annotation.

    The following two lines are examples extracted from training data:

        '''
        "微粒贷开通"	你好，我的微粒贷怎么没有开通呢	0
        为什么借款后一直没有给我回拨电话    怎么申请借款后没有打电话过来呢！    1
        '''

    Provider: Intelligent Computing Research Center, Harbin Institute of Technology(Shenzhen)
    Contacts: Qingcai Chen (email: qingcai.chen@hit.edu.cn; Fax: +86-755-26033182)
    More Info: https://www.luge.ai/

     """
```

Sorry I have been preparing for my final exam these days and I didn't reply timely. Here are my chages on this opinion.

Comment (smallv0221) [reply paddlenlp/datasets/bq_corpus.py:17]: Great description! Good luck in the final.

File: paddlenlp/datasets/bq_corpus.py (hunk 2 above)
Comment (dyan-dy) [reply paddlenlp/datasets/bq_corpus.py:26]: Sorry actually I've been trapped in the trouble for a long time and I don't know how to fix it ... I've been dealing with it for the past two days, and I can't solve the problem. Sorry for slowing down your progress : (

**Pull request merged**